odoo/addons/test_new_api/tests/test_one2many.py

```diff
@@ -239,6 +239,33 @@ def test_dont_write_the_existing_childs(self):
         a = parent.child_ids[0]
         parent.write({'child_ids': [Command.link(a.id), Command.create({'name': 'B'})]})
 
+    def test_create_with_commands(self):
+        child1 = self.env['test_new_api.order.line'].create({'product': 'set'})
+        child2 = self.env['test_new_api.order.line'].create({'product': 'sept'})
+
+        # warm up default_value
+        self.env['test_new_api.order'].create({
+            'line_ids': [Command.set(child1.ids)],
+        })
```

Comment:
You don't need to drop the constraint on `order_id` if you do this instead:
```suggestion
        # create lines and warm up caches
        order = self.env['test_new_api.order'].create({
            'line_ids': [Command.create({'product': name}) for name in ('set', 'sept')],
        })
        child1, child2 = order.line_ids
```